cmd/p2psim/main.go

```diff
@@ -132,6 +163,104 @@ func main() {
 							Value: "",
 							Usage: "node private key (hex encoded)",
 						},
+						&cli.BoolFlag{
+							Name:  "sim.dialer",
+							Usage: "Use the simulation dialer",
+						},
+						&cli.BoolFlag{
+							Name:  "fake.iplistener",
+							Usage: "Use the fake listener to random remote ip when accepting connections",
+						},
+						&cli.BoolFlag{
+							Name:  "start",
```

Comment:
default true

Reply:
hm I saw the example, they provided the start command, the flow should be 
```
p2p-sim node create 
p2p-sim node start 
```
could we follow this instead of opening new flag.

Reply:
I tried to add this flag to `p2psim node create` because` p2psim node create-multi` really needs it. Otherwise, we need to split it into two commands, `create-multi` and `start-multi` to make it clearer. 
If you think this is necessary, we can remove only the start flag in `p2psim node create`, keep the original, and `create-multi` will call starting nodes after successfully creating them.

Reply:
yup that's what I think keep the node create logic as original one. We can create command create-multi for utilizing our cases.